.circleci/sort-yaml.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
```

Comment:
I have added this little script to prove that my change yields identical output, modulo ordering of YAML keys.

Reply:
So this is a test file, right? How about adding it as `.circleci/test/test_sort_yaml.py` ?

Reply:
Will do, though it shouldn't be considered a standalone test, but rather a utility to facilitate a manual comparison for this somewhat extensive code change.